docs/contributing/contributing_test.md

```diff
@@ -17,6 +17,26 @@ For example, you can run `pytest --no-spark --no-sqlalchemy` to skip all local b
 
 Note: as of early 2020, the tests generate many warnings. Most of these are generated by dependencies (pandas, sqlalchemy, etc.) You can suppress them with pytest’s `--disable-pytest-warnings` flag: `pytest --no-spark --no-sqlalchemy --disable-pytest-warnings`.
 
+#### BigQuery tests
+
+In order to run BigQuery tests, you first need to go through the following steps:
+
+1. Select or create a Cloud Platform project.
+2. Setup Authentication.
+3. In your project, create a bigquery dataset named `test_ci` and set the dataset default table expiration to `.1` days
+
+* Select or create a Cloud Platform project.: [https://console.cloud.google.com/project](https://console.cloud.google.com/project)
+* Setup Authentication.: [https://googleapis.dev/python/google-api-core/latest/auth.html](https://googleapis.dev/python/google-api-core/latest/auth.html)
+* In your project, create a bigquery dataset named `test_ci`: [https://cloud.google.com/bigquery/docs/datasets](https://cloud.google.com/bigquery/docs/datasets)
+* Set the dataset default table expiration to `.1` days: [https://cloud.google.com/bigquery/docs/updating-datasets#table-expiration](https://cloud.google.com/bigquery/docs/updating-datasets#table-expiration)
+
+After setting up authentication, you can run with your project using the environment variable `GE_TEST_BIGQUERY_PROJECT`, e.g.
+
+```bash
+    GE_TEST_BIGQUERY_PROJECT=<YOUR_GOOGLE_CLOUD_PROJECT> 
+    pytest tests/test_definitions/test_expectations_cfe.py --bigquery --no-spark --no-postgresql
+```
+
```

Comment:
Super nitpicky but could we capitalize any usages of BigQuery in this file (and any other `.md`)?

Reply:
done!